verboseTree-maven-plugin/src/main/java/org/apache/maven/dependency/graph/DependencyGraphBuilder.java

```diff
@@ -39,12 +58,119 @@
     @Parameter( defaultValue = "${session}", readonly = true, required = true )
     protected MavenSession session;
 
+    @Parameter
+    private String outputFile;
+
+    @Component
+    private ProjectDependenciesResolver resolver;
+
+    // replace Component with sisu guice named or singleton annotation
     @Component
     private ArtifactHandlerManager artifactHandlerManager;
 
+    private SerializeGraph serializer;
+
+    private DependencyNode rootNode;
+
+    @Parameter( defaultValue = "${repositorySystemSession}" )
+    private RepositorySystemSession repositorySystemSession;
+
     public void execute() throws MojoExecutionException
     {
         getLog().info( project.getArtifactId() );
+        getLog().info( session.toString() );
+
+        try
+        {
+            rootNode = buildDependencyGraph();
+        }
+        catch ( Exception e )
```

Comment:
you should almost never have to catch a generic Exception. See Chapter 10 of Effective Java.